pyproject.toml

```diff
@@ -16,7 +16,7 @@
 
 [tool.poetry]
 name = "splunk_add_on_ucc_framework"
-version = "5.36.1"
+version = "5.36.2"
```

Comment:
curious why poetry.lock has not changed

Reply:
Release was connected just to docs error i made, so there was no need to install update code itself. It was just about releasing docs.